packages/graph-retriever/src/graph_retriever/testing/adapter_tests.py

```diff
@@ -253,21 +273,33 @@ class AdapterComplianceSuite(abc.ABC):
     loaded.
     """
 
-    def skip_case(self, method: str, case_id: str) -> None:
+    def expected(self, method: str, case: AdapterComplianceCase) -> list[str]:
```

Comment:
The main change here is replacing the "skip if you want to" hook, with the ability for the test class to override the expecations. As noted, we generally *shouldn't* do this (especially since if the case names change, etc. there will be drift). However, it is useful in a case like this where the embeddings are different.